docs/status.md

```diff
@@ -17,12 +17,11 @@ Filing [issues](https://github.com/shorebirdtech/shorebird/issues) is a good way
 to provide feedback. Feedback via [Discord](https://discord.gg/shorebird) is
 also welcome.
 
-Our guiding principle for these early days is "first, do no harm". It should be
-the case that using Shorebird is never worse than not using Shorebird. It is
-still possible using early versions of Shorebird could break your app in the
-wild. However thousands of apps have already tried Shorebird this year and we
-have only ever seen one break in the wild (a bad interaction with a third party
-library). Please let us know if you encounter any issues.
+Our guiding principle has been "first, do no harm". It should be
+the case that using Shorebird is never worse than not using Shorebird. Of the
+thousands of apps have already tried Shorebird this year we're only aware
+of two having seen issues with Shorebird in the wild (one with a bad interaction
```

Comment:
Is this referencing the Firebase messaging issue? "a bad interaction with a third party library" sounds kind of scary – more context or some steps we've taken to mitigate would be reassuring here.

Reply:
I think I'll jsut remove it.  The two crashes we know of are the FCM issue and the iOS alpha crasher.

Reply:
Fixed.